### PR TITLE
Adding python3.X runtime on Jenkins

### PR DESCRIPTION
**Pull request by Johnbush**
**Description of the Change**

- Added Installing scripts for python3.X and it's dependencies

- Setting alias for both python and python3

### CODE REVIEW

File: installscripts/dockerfiles/jenkins-ce/Dockerfile [hunk 1]
```diff
@@ -5,9 +5,13 @@ MAINTAINER JazzOSS Team
 # Switching to root to configure the image with system packages
 USER root
 RUN apt-get update && apt-get install -y vim curl sudo libtool autoconf make unzip rsync gcc autogen shtool pkg-config lsb-release python python-dev python-pip python-setuptools groff less && \
-curl https://bootstrap.pypa.io/get-pip.py | python && pip install --upgrade awscli && apt-get clean && pip install virtualenv && /usr/bin/easy_install virtualenv
+python3.6 python3-pip build-essential libssl-dev libffi-dev python3-dev && curl https://bootstrap.pypa.io/get-pip.py | python && curl https://bootstrap.pypa.io/get-pip.py | python3 && pip install --upgrade awscli && \
+pip3 install --upgrade awscli  apt-get clean && pip install virtualenv && pip3 install virtualenv && /usr/bin/easy_install virtualenv
```
Comment (svidhani) [inline installscripts/dockerfiles/jenkins-ce/Dockerfile]: is upgrade awscli needed?
also why is it in above line as well?

Comment (Johnbush) [reply installscripts/dockerfiles/jenkins-ce/Dockerfile]: It is not required and removed it.

**Comment by bleggett** [comment]
Can this please be retargeted against the `jenkins_docker` branch, which is where related work is happening.

**Comment by svidhani** [comment]
@bleggett I have merged jenkins_docker into python3. We want to keep the branches clean and on jazz side we have a python3 branch. So this way we can have 1:1 mapping and can validate in the stack. Let me know what you think

**Comment by bleggett** [comment]
@svidhani It's not exactly _clean_ to include a mishmash of release features in an arbitrary feature branch, but it's how it has to be since our CI process isn't particularly robust.

**Comment by devsatishm** [comment]
@bleggett Actually, goal for this PR is different from `jenkins_docker` work. I think @svidhani needed changes from `jenkins_docker` to  work on this and so pulled changes from it. I think it wouldn't have been necessary if these changes are in `develop` already. 

Going forward, we'll branch out from `develop` & merge changes back to `develop` as we discussed.

**Pull request merged**